container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -459,7 +491,7 @@ instance_groups:
                       values:
                       - diego-api
                   topologyKey: "beta.kubernetes.io/os"
-          service-account: withsysresource
+          service-account: privileged
```

Comment:
If we do `service-account` for BPM instead, we can then not touch this (and have it be merged to be `privileged`, at least in theory)

Reply:
Going to a separate SA likely requires a full round of re-testing.
I see how it would be a good thing, I also shudder at the above re-test.

Reply:
I'm okay with not doing that for now

Reply:
Ref https://trello.com/c/gEuXlu5g/627-give-bpm-job-its-own-service-account